MySQL CI: TestConvergenceAndAccuracy fails when using the refactorized codes

Based on https://github.com/sql-machine-learning/sqlflow/pull/2970 .

```
2020-09-30T01:48:08.3378575Z [0m[91m--- FAIL: TestConvergenceAndAccuracy (12.46s)
2020-09-30T01:48:08.3380698Z [0m[0m    convergence_test.go:55: 
2020-09-30T01:48:08.3381406Z [0m[0m        	Error Trace:	convergence_test.go:55
2020-09-30T01:48:08.3381948Z [0m[0m        	            				convergence_test.go:42
2020-09-30T01:48:08.3382456Z [0m[0m        	Error:      	Should be true
2020-09-30T01:48:08.3383159Z [0m[0m        	Test:       	TestConvergenceAndAccuracy
2020-09-30T01:48:08.3383887Z [0m[0m        	Messages:   	runSQLProgram error: exit status 1
2020-09-30T01:48:08.3384508Z [0m[0m        	            	Traceback (most recent call last):
2020-09-30T01:48:08.3385059Z [0m[0m        	            	  File "<stdin>", line 27, in <module>
2020-09-30T01:48:08.3385977Z [0m[0m        	            	  File "/home/sqlflow/actions-runner/_work/sqlflow/sqlflow/python/runtime/local/submitter.py", line 95, in submit_local_train
2020-09-30T01:48:08.3386736Z [0m[0m        	            	    load=load)
2020-09-30T01:48:08.3388123Z [0m[0m        	            	  File "/home/sqlflow/actions-runner/_work/sqlflow/sqlflow/python/runtime/local/tensorflow_submitter/train.py", line 42, in train
2020-09-30T01:48:08.3389479Z [0m[0m        	            	    load=load)
2020-09-30T01:48:08.3390561Z [0m[0m        	            	  File "/home/sqlflow/actions-runner/_work/sqlflow/sqlflow/python/runtime/step/tensorflow/train.py", line 98, in train_step
2020-09-30T01:48:08.3391417Z [0m[0m        	            	    model_params_constructed[optimizer_arg])
2020-09-30T01:48:08.3392373Z [0m[0m        	            	  File "/home/sqlflow/actions-runner/_work/sqlflow/sqlflow/python/runtime/step/tensorflow/train.py", line 39, in get_tf_optimizer
2020-09-30T01:48:08.3393555Z [0m[0m        	            	    return getattr(tf_optimizers, optimizer)
2020-09-30T01:48:08.3394459Z [0m[0m        	            	AttributeError: module 'tensorflow_core.keras.optimizers' has no attribute 'Adam()'
2020-09-30T01:48:08.3395146Z [0m[0m        	            	: 
2020-09-30T01:48:08.3395593Z [0m[0m    convergence_test.go:63: 
2020-09-30T01:48:08.3396151Z [0m[0m        	Error Trace:	convergence_test.go:63
2020-09-30T01:48:08.3396925Z [0m[0m        	            				convergence_test.go:57
2020-09-30T01:48:08.3397744Z [0m[0m        	Error:      	Should be true
2020-09-30T01:48:08.3398762Z [0m[0m        	Test:       	TestConvergenceAndAccuracy
2020-09-30T01:48:08.3400059Z [0m[0m        	Messages:   	runSQLProgram error: exit status 1
2020-09-30T01:48:08.3400872Z [0m[0m        	            	Traceback (most recent call last):
2020-09-30T01:48:08.3401426Z [0m[0m        	            	  File "<stdin>", line 13, in <module>
2020-09-30T01:48:08.3402333Z [0m[0m        	            	  File "/home/sqlflow/actions-runner/_work/sqlflow/sqlflow/python/runtime/local/submitter.py", line 116, in submit_local_pred
2020-09-30T01:48:08.3403171Z [0m[0m        	            	    model=model)
2020-09-30T01:48:08.3404050Z [0m[0m        	            	  File "/home/sqlflow/actions-runner/_work/sqlflow/sqlflow/python/runtime/local/tensorflow_submitter/predict.py", line 88, in pred
2020-09-30T01:48:08.3404936Z [0m[0m        	            	    predict_generator, selected_cols)
2020-09-30T01:48:08.3405888Z [0m[0m        	            	  File "/home/sqlflow/actions-runner/_work/sqlflow/sqlflow/python/runtime/tensorflow/predict.py", line 229, in estimator_predict
2020-09-30T01:48:08.3406695Z [0m[0m        	            	    is_xgboost=False)
2020-09-30T01:48:08.3407531Z [0m[0m        	            	  File "/home/sqlflow/actions-runner/_work/sqlflow/sqlflow/python/runtime/db.py", line 208, in read_features_from_row
2020-09-30T01:48:08.3408366Z [0m[0m        	            	    feature = read_feature(row[select_cols.index(name)],
2020-09-30T01:48:08.3409014Z [0m[0m        	            	ValueError: 'news_title' is not in list
2020-09-30T01:48:08.3409638Z [0m[0m        	            	: Start predicting using estimator model...
2020-09-30T01:48:08.3410205Z [0m[0m    convergence_test.go:67: 
2020-09-30T01:48:08.3410975Z [0m[0m        	Error Trace:	convergence_test.go:67
2020-09-30T01:48:08.3411535Z [0m[0m        	            				convergence_test.go:65
2020-09-30T01:48:08.3412088Z [0m[0m        	Error:      	Received unexpected error:
2020-09-30T01:48:08.3412806Z [0m[0m        	            	Error 1146: Table 'sanity_check.predict' doesn't exist
2020-09-30T01:48:08.3413540Z [0m[0m        	Test:       	TestConvergenceAndAccuracy
2020-09-30T01:48:08.3414145Z [0m[0m    convergence_test.go:65: 
2020-09-30T01:48:08.3414695Z [0m[0m        	Error Trace:	convergence_test.go:65
2020-09-30T01:48:08.3415408Z [0m[0m        	Error:      	func (assert.PanicTestFunc)(0xcabd40) should not panic
2020-09-30T01:48:08.3416266Z [0m[0m        	            		Panic value:	runtime error: invalid memory address or nil pointer dereference
2020-09-30T01:48:08.3417061Z [0m[0m        	Test:       	TestConvergenceAndAccuracy
```

--- a/python/runtime/step/tensorflow/train.py
+++ b/python/runtime/step/tensorflow/train.py
@@ -14,7 +14,7 @@
 import copy
 import types
 
-import tensorflow.keras.losses as tf_loss
+import tensorflow.keras.losses as tf_loss  # noqa: F401
 from runtime import db
 from runtime.feature.compile import compile_ir_feature_columns
 from runtime.feature.derivation import (get_ordered_field_descs,
@@ -30,17 +30,17 @@ from runtime.tensorflow.input_fn import get_dataset_fn
 from runtime.tensorflow.set_log_level import set_log_level
 
 if tf_is_version2():
-    import tensorflow.keras.optimizers as tf_optimizers
+    import tensorflow.keras.optimizers as tf_optimizers  # noqa: F401
 else:
-    import tensorflow.train as tf_optimizers
+    import tensorflow.train as tf_optimizers  # noqa: F401
 
 
 def get_tf_optimizer(optimizer):
-    return getattr(tf_optimizers, optimizer)
+    return eval("tf_optimizers." + optimizer)
 
 
 def get_tf_loss(loss):
-    return getattr(tf_loss, loss)
+    return eval("tf_loss." + loss)
 
 
 # NOTE(typhoonzero): workflow step entry for codegen/experimental,

